src/battlesnake.theserverproject.com/insert_clever_name/snake/game/game.go

```diff
@@ -1,18 +1,20 @@
 package game
 
 const (
-	WALL  int = -1
-	EMPTY int = 0
-	FOOD  int = 1
+	WALL    int = -1
```

Comment:
Go has some support for Enums it would seem (https://yourbasic.org/golang/iota/). Might be worth changing these to use enums?

Reply:
Sure, can do it that way